security-proxy/src/main/java/org/georchestra/security/Proxy.java

```diff
@@ -709,8 +709,15 @@ private void handleRequest(HttpServletRequest request, HttpServletResponse final
             doHandleRequest(finalResponse, proxiedResponse);
         } catch (IOException | ExecutionException | InterruptedException | TimeoutException e) {
             // connection problem with the host
-            logger.error("Exception occured when trying to connect to the remote host: ", e);
+            String errMsg = String.format("Exception occured when trying to connect to the remote url '%s'", sURL);
+            logger.error(errMsg, e);
```

Comment:
I don't know why there is a debug(format, arg1, arg2, ...) and a debug(msg, throwable), and no debug(format, arg1, arg2, ..., throwable), but there is maybe a good reason.

Reply:
don't know either